configure.ac

```diff
@@ -1,6 +1,11 @@
     AC_INIT([suricata],[6.0.1-dev])
     m4_ifndef([AM_SILENT_RULES], [m4_define([AM_SILENT_RULES],[])])AM_SILENT_RULES([yes])
     AC_CONFIG_HEADERS([src/autoconf.h])
+
+    # FIXME: Rename HAVE_CONFIG_H to HAVE_AUTOCONF_H
```

Comment:
fixme

Reply:
done

Reply:
> fixme

@victor : here

Reply:
nope. You probably meant @victorjulien ?